### PR TITLE
Drop legacy python

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 envlist =
     flake8,
-    py27,
-    py36,
     py37,
     py38,
+    py39,
+    py310,
+    py311,
 minversion = 1.6
```
Comment (sallner) [inline tox.ini]: Do we need version 4 here?

**Review by sallner — APPROVED** [review]
Looks good for me.

File: tox.ini (hunk 1 above)
Comment (icemac) [reply tox.ini]: We are not forced to, but I'll update.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**